ci/scripts/PKGBUILD

```diff
@@ -116,6 +119,7 @@ build() {
     -DARROW_WITH_BZ2=ON \
     -DARROW_ZSTD_USE_SHARED=OFF \
     -DARROW_CXXFLAGS="${CPPFLAGS}" \
+    -DAWSSDK_SOURCE=BUNDLED \
```

Comment:
Can we add a comment explaining why the MinGW package doesn't work?

Reply:
I've added a comment on https://github.com/apache/arrow/pull/44989/commits/da4e777f2a091248d4f1de44546b8b5e97a09c4e

Reply:
I also added a readme to the patches dir: f17cb8fa557c3f9de9863b8e89aaf72b45e38358.